drivers/adc/ad9081/ad9081.c

```diff
@@ -707,18 +707,37 @@ static int32_t ad9081_setup(struct ad9081_phy *phy)
 	mdelay(10);
 
 	if (phy->jesd_rx_clk) {
-		mdelay(10);
-		ret = clk_enable(phy->jesd_rx_clk);
-		if (ret < 0) {
+		int timeout = 2000;
+		while(timeout) {
+			ret = clk_enable(phy->jesd_rx_clk);
+			if (ret) {
```

Comment:
Can the changes in this file be in a separate commit?
"Add timeout for clk_enable " or something like that

Reply:
Yeah good point